libs/execution/include/hpx/execution/dataflow.hpp

```diff
@@ -192,21 +174,23 @@ namespace hpx { namespace lcos { namespace detail
             // simply schedule new thread
             parallel::execution::parallel_policy_executor<launch::async_policy>
                 exec{policy};
-            parallel::execution::post(exec,
+            parallel::execution::post(
+                exec,
                 [this_ = std::move(this_)](Futures&& futures) -> void {
                     return this_->done(std::move(futures));
-                }, std::move(futures));
+                },
+                std::move(futures));
```

Comment:
Same here, since we know that `parallel_policy_executor<launch::async_policy>` directly implements a `post` method, we can save us going through the customization point and call it directly on the executor instead.